### ISSUE TITLE
Feature request: flac support

### ISSUE THREAD

**Issue by remenor** [thp/wavbreaker#23]
The software is great. 
Is it possible to add FLAC file reading and writing support?
Sorry for my english

**Comment by thp** [comment]
Yes, it's possible, but not implemented yet. As FLAC is lossless, you can decode the FLAC file, cut the WAV file and then re-encode to FLAC without any loss for now.

It might be convenient to skip the decoding/encoding step for cutting, but it's not as high priority as cutting lossy audio (as there decoding, cutting the WAV and re-encoding introduces generational loss).

**Comment by remenor** [comment]
I understand. Since it's not a priority (yet) is it possible to add support for the specification [WAVEFORMATEXTENSIBLE](https://learn.microsoft.com/en-us/windows/desktop/api/mmreg/ns-mmreg-waveformatextensible)?
I use it to cut demultiplexed audio with ffmpeg. If the wav format is used and the sample size is greater than 16 bits, wavbreaker cannot interpret it correctly. Apparently it can only open files with the legacy specification ([PCMWAVEFORMAT](https://learn.microsoft.com/en-us/windows/win32/api/mmeapi/ns-mmeapi-pcmwaveformat))

**Comment by thp** [comment]
> I understand. Since it's not a priority (yet) is it possible to add support for the specification [WAVEFORMATEXTENSIBLE](https://learn.microsoft.com/en-us/windows/desktop/api/mmreg/ns-mmreg-waveformatextensible)?
> I use it to cut demultiplexed audio with ffmpeg. If the wav format is used and the sample size is greater than 16 bits, wavbreaker cannot interpret it correctly. Apparently it can only open files with the legacy specification ([PCMWAVEFORMAT](https://learn.microsoft.com/en-us/windows/win32/api/mmeapi/ns-mmeapi-pcmwaveformat))

Please file a separate issue for this one, it's separate from FLAC support.

**Comment by remenor** [comment]
Done
